src/Constants.iOS.cs.in

```diff
@@ -135,5 +135,7 @@ namespace MonoTouch {
 		public const string CarPlayLibrary = "/System/Library/Frameworks/CarPlay.framework/CarPlay";
 		public const string IdentityLookupUILibrary = "/System/Library/Frameworks/IdentityLookupUI.framework/IdentityLookupUI";
 		public const string NaturalLanguageLibrary = "/System/Library/Frameworks/NaturalLanguage.framework/NaturalLanguage";
+		public const string SoundAnalysisLibrary = "/System/Library/Frameworks/SoundAnalysis.framework/SoundAnalysis";
```

Comment:
Adding an iOS 13 comment just above would make it easier (to match the rest of the code)

```csharp
// iOS 13.0
public const string SoundAnalysisLibrary ...
```